core/templates/domain/exploration/StatesObjectFactory.ts

```diff
@@ -147,17 +141,15 @@ export class States {
   providedIn: 'root'
 })
 export class StatesObjectFactory {
-  constructor(private stateObject: StateObjectFactory) {}
-  // TODO(ankita240796): Remove the bracket notation once Angular2 gets in.
-  /* eslint-disable dot-notation */
-  createFromBackendDict(statesBackendDict: any): States {
-  /* eslint-enable dot-notation */
-    var stateObjectsDict = {};
-    for (var stateName in statesBackendDict) {
-      stateObjectsDict[stateName] = this.stateObject.createFromBackendDict(
-        stateName, statesBackendDict[stateName]);
+  constructor(private stateObjectFactory: StateObjectFactory) {}
```

Comment:
Just to check, has to TODO above been addressed as part of this PR? If not, maybe we should keep it.

Reply:
Hadn't even noticed, yup you're right looks like the comment is addressed.